ext/calendar/easter.c

```diff
@@ -28,6 +28,7 @@ static void _cal_easter(INTERNAL_FUNCTION_PARAMETERS, bool gm)
 	struct tm te;
 	zend_long year, golden, solar, lunar, pfm, dom, tmp, easter, result;
 	zend_long method = CAL_EASTER_DEFAULT;
+	const double max_year = (double)(ZEND_LONG_MAX / 1.25);
```

Comment:
The double cast is not necessary.

Reply:
How did you come to the conclusion to divide by 1.25?

Reply:
because year + (year/4) = 1.25*year;

Reply:
Ah, and because `- (year/100) + (year/400)` must be less than 0 this is indeed sufficient of course. This is right.